eclipse-plugins/org.uberfire.eclipse.runtime/pom.xml

```diff
@@ -0,0 +1,131 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.uberfire.eclipse</groupId>
+		<artifactId>org.uberfire.eclipse</artifactId>
+		<version>1.0.0-SNAPSHOT</version>
+	</parent>
+	<artifactId>org.uberfire.eclipse.runtime</artifactId>
+	<packaging>bundle</packaging>
+
+	<name>Uberfire-Eclipse : Embedded runtime</name>
+	<properties>
+		<osgi-version-qualifier>${BUILD_ALIAS}-v${maven.build.timestamp}-B${BUILD_NUMBER}</osgi-version-qualifier>
+	</properties>
+	<dependencies>
+		<!-- Uberfire vvv -->
+		<dependency>
+			<groupId>org.uberfire</groupId>
+			<artifactId>uberfire-api</artifactId>
+			<version>1.0.0-SNAPSHOT</version>
+		</dependency>
+		<!-- Uberfire ^^^ -->
+
+		<!-- Drools vvv -->
+		<dependency>
+			<groupId>org.drools</groupId>
+			<artifactId>drools-wb-drl-text-editor-api</artifactId>
+			<version>7.0.0-SNAPSHOT</version>
+		</dependency>
+		<dependency>
+			<groupId>org.drools</groupId>
+			<artifactId>drools-workbench-models-commons</artifactId>
+			<version>7.0.0-SNAPSHOT</version>
+		</dependency>
+		<dependency>
+			<groupId>org.drools</groupId>
+			<artifactId>drools-workbench-models-datamodel-api</artifactId>
+			<version>7.0.0-SNAPSHOT</version>
+		</dependency>
+		<!-- Drools ^^^ -->
+	</dependencies>
+	<build>
+		<plugins>
+			<plugin>
+				<groupId>org.eclipse.tycho</groupId>
+				<artifactId>tycho-maven-plugin</artifactId>
+				<extensions>true</extensions>
+			</plugin>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-compiler-plugin</artifactId>
+				<version>3.1</version>
+				<configuration>
+					<source>1.8</source>
+					<target>1.8</target>
+				</configuration>
+			</plugin>
+			<plugin>
+				<groupId>org.apache.felix</groupId>
+				<artifactId>maven-bundle-plugin</artifactId>
+				<version>2.5.0</version>
+				<extensions>true</extensions>
+				<configuration>
+					<instructions>
+						<Embed-Transitive>true</Embed-Transitive>
+						<_failok>true</_failok>
+						<_nouses>true</_nouses>
+						<_nodefaultversion>true</_nodefaultversion>
+						<_snapshot>${osgi-version-qualifier}</_snapshot>
+						<Bundle-SymbolicName>${project.artifactId};singleton:=false</Bundle-SymbolicName>
+						<Bundle-RequiredExecutionEnvironment>JavaSE-1.8</Bundle-RequiredExecutionEnvironment>
+						<Bundle-Name>%Bundle-Name</Bundle-Name>
+						<Bundle-Vendor>%Bundle-Vendor</Bundle-Vendor>
+						<Bundle-ClassPath>{maven-dependencies}</Bundle-ClassPath>
+						<Eclipse-BundleShape>dir</Eclipse-BundleShape>
+						<archive>
+							<addMavenDescriptor>false</addMavenDescriptor>
+						</archive>
+						<Embed-Dependency>
+							*;scope=compile|runtime;artifactId=!slf4j-api|javax.inject|
```

Comment:
there's most certainly some cleanup to do here, you probably don't want to embed all the transitive dependencies. I'll leave that to the future maintainer.